### PR TITLE
[FutureCCXTExchange] Add default implementations

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot_trading/exchanges/implementations/future_ccxt_exchange.py [hunk 1]
```diff
@@ -167,3 +171,58 @@ def clean_trade(self, trade):
...
+
+    def parse_position(self, position_dict) -> dict:
+        try:
+            position_dict.update({
+                trading_enums.ExchangeConstantsPositionColumns.SYMBOL.value:
+                    position_dict.get(position_dict[trading_enums.ExchangePositionCCXTColumns.SYMBOL.value]),
+                trading_enums.ExchangeConstantsPositionColumns.ID.value:
+                    position_dict.get(position_dict[trading_enums.ExchangePositionCCXTColumns.SYMBOL.value]),
+                trading_enums.ExchangeConstantsPositionColumns.QUANTITY.value:
+                    position_dict.get(trading_enums.ExchangePositionCCXTColumns.CONTRACT_SIZE.value, constants.ZERO),
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/implementations/future_ccxt_exchange.py:199]: no need to cast it into decimal.Decimal here ?

Comment (Herklos) [reply octobot_trading/exchanges/implementations/future_ccxt_exchange.py:199]: Yes we need a cast

File: octobot_trading/exchanges/implementations/future_ccxt_exchange.py [hunk 2]
```diff
@@ -167,3 +168,66 @@ def clean_trade(self, trade):
...
+                    decimal.Decimal(
+                        position_dict.get(trading_enums.ExchangePositionCCXTColumns.CONTRACT_SIZE.value, 0)),
+                trading_enums.ExchangeConstantsPositionColumns.COLLATERAL.value:
+                    decimal.Decimal(
+                        position_dict.get(trading_enums.ExchangePositionCCXTColumns.COLLATERAL.value, 0)),
+                trading_enums.ExchangeConstantsPositionColumns.NOTIONAL.value:
+                    decimal.Decimal(
+                        position_dict.get(trading_enums.ExchangePositionCCXTColumns.NOTIONAL.value, 0)),
+                trading_enums.ExchangeConstantsPositionColumns.LEVERAGE.value:
+                    decimal.Decimal(
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/implementations/future_ccxt_exchange.py:213]: We might need to do `f"{position_dict.get(trading_enums.ExchangePositionCCXTColumns.LEVERAGE.value, 0)}"` to ensure no rounding issues (creating a decimal from float can add .99999999 etc numbers, it doesn't happen from str)

Comment (Herklos) [reply octobot_trading/exchanges/implementations/future_ccxt_exchange.py:213]: Right :+1:

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**